### ISSUE TITLE
Draft: uniform scaling option for Scale and Clone

### ISSUE THREAD

**Issue by FEA-eng** [FreeCAD/FreeCAD#13324]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

Draft Scale and Clone tools could significantly benefit from having an option to apply uniform scaling (specify a single scale factor). Currently, it's necessary to enter the scale factors one by one and wait for the model to rebuild 3 times (which may take a while in the case of larger models). I know that a workaround could be to use Part Scale (which has the uniform scaling option) but it would still be nice to have uniform scaling available in Draft.

### Full version info

```shell
OS: Windows 10 build 19045
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.36731 (Git)
Build type: Release
Branch: main
Hash: ac1d887336826c59321352f49a195bf1ef8be2f0
Python 3.11.8, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: Polish/Poland (pl_PL)
```

### Subproject(s) affected?

Draft

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Roy-043** [comment]
We need to distinguish the tool and the object here. The issue is about the Draft_Clone object. The Draft_Scale tool does have a uniform scale option (a checkbox).

There is no consistent way to handle scaling properties across FreeCAD:
* Draft_Clone uses 1 vector.
* Part_Scale has 4 inputs, 1 for uniform scaling and 3 for non-uniform.
* Link has 2 inputs, 1 for uniform scale and a vector for non-uniform scaling (you have to check "Show all" in the tree for the vector input).

One thing that might be implemented is to open a scaling dialog if a Draft_Clone is double-clicked in the Tree view.

**Comment by FEA-eng** [comment]
> One thing that might be implemented is to open a scaling dialog if a Draft_Clone is double-clicked in the Tree view.

Right, that would be nice. I updated the issue following your feedback.

**Comment by yorikvanhavre** [comment]
I would do the same as App::Link I guess... A  visible, uniform scaling prop and a hidden vector one for the few who need differential scaling. The problem is what updates what between these two... Should the uniform scaling "reset" the vector scaling prop when changed?

**Comment by yorikvanhavre** [comment]
But then you have the risk to see your vector scaling values accidentally reset (for ex when loading a file). Maybe one way to solve this could be to say "if you want to use vector scaling, you must set uniform scaling to zero"

**Comment by FEA-eng** [comment]
Or maybe the vector scaling fields could be disabled until a checkbox or true/false preference "Use non-uniform scaling" is enabled. Then the uniform scale field would become grayed out.
